dev/com.ibm.ws.javaee.platform.v8/build.xml

```diff
@@ -0,0 +1,13 @@
+<!--
```

Comment:
this file can be removed, along with the build.xml file from the javaee.platform.v7 project you copied it from (build.xml files are not used in OpenLiberty)